modules/cli/pages/cbsummary.adoc

```diff
@@ -0,0 +1,56 @@
+= cbsummary
+:page-topic-type: reference
+
+[abstract]
+The [.cmd]cbsummary tool connects to a set of Couchbase clusters and generates a summary report.
+
+== Description
+
+The [.cmd]cbsummary tool connects to a set of Couchbase clusters specified in a configuration file and generates a summary report. 
+
+The config file contains JSON specifying an array of information on each cluster, giving the Couchbase credentials to login, and one or more IP addresses for cluster nodes.
+
+The default report format includes RAM and core utilization across each specified cluster, since that information is useful in determining compliance with Couchbase licenses. If you specify --csv, then the report is generated in CSV instead of JSON. If, instead, you specify --full, then a much more detailed report is generated.
+
+The summary report is sent to the file 'cbsummary.out.<timestamp>', unless a different file name is specified with the --output option.
+
+== Syntax
+
+The syntax for `cbsummary` is as follows:
+
+----
+cbsummary --config=<config file> [--output=<output file>] [--full]
+----
+
+== Options
+
+Command options are as follows:
+
+[cols="3,3"]
+|===
+| Option | Description
+
+| `--config [config file]`
+| Configuration file listing clusters and credentials to connect and generate a summary.
+
+| `--csv`
+| Generate the report in CSV format. Not that this format is not compatible with full reports.
+
+| `--full`
+| Generate an extensive report instead of just core and RAM usage.
+
+| `-h --help`
+| Print a help message and exit.
+
+| `--output [output file]`
+| Name of the output file. The default naming convention for the report is `cbsummary.out.<timestamp>`.
+|===
+
+== Examples
```

Comment:
Usually I would expect the examples section to include an example of the command be used, perhaps with more complex usages of syntax and options

Reply:
Agree, but in this case, the basic example was good to put this help page on the site.